src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor

```diff
@@ -30,19 +36,36 @@
                       title="@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsMessageFilter)]"
                       slot="end" />
         <FluentDivider slot="end" Role="DividerRole.Presentation" Orientation="Orientation.Vertical" />
-        <div title="@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsMinimumLogFilter)]" slot="end" style="display: flex;align-items: center;">
-            <span>@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsLevels)]</span>
-            <FluentDivider Role="DividerRole.Presentation" Orientation="Orientation.Vertical" />
+        @if (ViewportInformation.IsDesktop)
+        {
+            <div title="@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsMinimumLogFilter)]" slot="end" style="display: flex;align-items: center;">
+                <span>@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsLevels)]</span>
+                <FluentDivider Role="DividerRole.Presentation" Orientation="Orientation.Vertical"/>
+
+                <FluentSelect TOption="SelectViewModel<LogLevel?>"
+                              Items="@_logLevels"
+                              Position="SelectPosition.Below"
+                              OptionText="@(c => c.Name)"
+                              @bind-SelectedOption="PageViewModel.SelectedLogLevel"
+                              @bind-SelectedOption:after="HandleSelectedLogLevelChangedAsync"
+                              Width="120px"
+                              Style="min-width: auto;"
+                              AriaLabel="@Loc[nameof(Dashboard.Resources.StructuredLogs.StructuredLogsSelectMinimumLogLevel)]"/>
```

Comment:
Can this duplication be removed? This sort of copy and paste feels like a bug waiting to happen. For example, one drop down is changed, and the other isn't. This goes for most duplication in this PR.

Reply:
Made this one component now